pkg/csi/manila/sanity/fakecsiclient.go

```diff
@@ -0,0 +1,85 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package sanity
+
+import (
+	"context"
+	"time"
+
+	"github.com/container-storage-interface/spec/lib/go/csi"
+	"google.golang.org/grpc"
+	"k8s.io/cloud-provider-openstack/pkg/csi/manila/csiclient"
+)
+
+type fakeIdentitySvcClient struct{}
+
+func (c fakeIdentitySvcClient) GetPluginInfo(context.Context) (*csi.GetPluginInfoResponse, error) {
```

Comment:
just curious...why you need to `fakeIdentitySvcClient`,  `fakeNodeSvcClient`?? sanity tests needs to fake cloud/mount functionality only, not nodeserver and identity server methods?

Reply:
CSI Manila takes on a bit different approach than Cinder. All node-related operations are delegated to, and performed by a dedicated Node Plugin (see the docs [here](https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/using-manila-csi-plugin.md#deployment)).